.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "master"
+    ]
+}
```

Comment:
@Qifull This VSCode configuration is necessary to add, isn’t it?

Reply:
Sorry, my mistake. Made so many changes at once, didn’t have time to react.